### DISCUSSION TITLE
Heat of hydration

### DISCUSSION THREAD

**Discussion by bimintelligence** [xcfem/xc#118] [General]
Hi everyone! I'm looking for a method to calculate the heat of hydration.
How can xc to calculate it...

**Comment by lcpt** [comment]
Hi, @SHAWNZHANGSH 

You mean the heat of hydration of cement in concrete, I imagine. Anyway, I don't think you can do this with XC, unless you find a suitable way to express the heat equation in terms of strains, stresses, stiffness, etc. I don't think this is a good idea.

You can try other open source finite element packages like Elmer or Code-Aster.

**Reply by bimintelligence** [reply]
Got it, thanks

**Comment by lcpt** [comment]
With pleasure!
